web3js-ext/src/web3/klaytn_tx.ts

```diff
@@ -131,7 +139,18 @@ export class KlaytnTx extends LegacyTransaction {
       data:     bytesToHex(this.data),
       input:    bytesToHex(this.data),
       chainId:  this.chainId ? toHex(this.chainId) : undefined,
-    });
+      humanReadable: false, 
+      codeFormat: 0x00,
+      key: this.key,
+    }; 
+
+    if ( txData.type == 0x28 ) {
```

Comment:
Use the TxType enum please

Reply:
Good, I checked https://github.com/klaytn/web3klaytn/pull/507/commits/076027bbf9c1532934985aa561100b15b6df0c54